internal/kokoro/synth/synth.sh

```diff
@@ -9,11 +9,10 @@ set -e -x
 cd $(dirname $0)/../../..
 
 # Install Go 1.15
-rm -rf /usr/local/go
 curl -o /tmp/go.tgz https://dl.google.com/go/go1.15.1.linux-amd64.tar.gz &&
-    tar -C /usr/local -xzf /tmp/go.tgz &&
+    tar -C $HOME -xzf /tmp/go.tgz &&
```

Comment:
Will this mess with the default GOPATH or anything? Might be better to use another directory.

Reply:
That is a very good point.... best error on the safe side.